package.json

```diff
@@ -94,6 +94,7 @@
     "jest": "23.4.1",
     "node-fetch": "^2.1.2",
     "patch-package": "^5.1.1",
+    "postinstall-prepare": "^1.0.1",
```

Comment:
why this?

Reply:
https://www.npmjs.com/package/patch-package#why-use-postinstall-prepare-with-yarn

Reply:
yes I understand what it does, I mean why you need it to fix this particular issue with push notifications

Reply:
Because I used `patch-package` to create the patch. `postinstall-prepare` is required by `patch-package` when using `parn` in order to apply the patch.